CONTRIBUTING.md

```diff
@@ -23,6 +23,10 @@ Pull requests are the best way to propose changes to the codebase (we use [Githu
 5. Make sure your code lints.
 6. Issue that pull request!
 
+## We Use Module Path Aliases
+
+Look at [Next.js Documention](https://nextjs.org/docs/advanced-features/module-path-aliases) and the `jsconfig.js` file for further instuctions on absolute imports and module path aliases. Please keep this in mind if you need to add imports to make changes.
```

Comment:
Please add examples and explanations on how this works. We are a learning community, so we have to teach as well thank you 👍